requests/cutlass.yaml

```diff
@@ -0,0 +1,8 @@
+action: cirun
+feedstocks:
+  - cutlass
+resources:
+  - cirun-openstack-gpu-large
+  - cirun-openstack-cpu-large
```

Comment:
I don't actually think we'll need the CPU version, as cutlass only has GPU builds.

Reply:
The GPU I do not think its actually needed. We are going to compile but not test.
We could test on the V100 available in the self-hosted runner, but I believe it is a waste to hold the GPU during 24h just for that...

Reply:
In that sense cutlass would benefit from ci_4xlarge, which has 8 cores and a lot of RAM

Reply:
> We could test on the V100 available in the self-hosted runner, but I believe it is a waste to hold the GPU during 24h just for that...

The openstack runner is _much_ more powerful than the generic azure agents. I think if the pytorch megabuild runs through with `cirun-openstack-gpu-large`, it's unlikely we'll need something bigger for cutlass. But on that I'll defer to @jaimergp and @isuruf

Reply:
Let's try with `large` first and if it proves to be insufficient, we can check xlarge. Agree that there's no GPU required in this case.